Makefile

```diff
@@ -0,0 +1,32 @@
+# Variables
+BINARY_NAME=thunder
+OUTPUT_DIR=target
+ZIP_FILE=product-is-thunder.zip
+REPOSITORY_DIR=cmd/server/repository
+VERSION_FILE=version.txt
```

Comment:
Instead of keeping version in a text file, shall we take the latest version from git tags? maintaining a version in text file is unnecessary and can lead to issues in the long run IMO

Reply:
That way we'll not be able to keep the product version tracked in the repository and the zip distribution unless rely on git actions/ jenkins builds. IMO this approach is better.
This is the approach we've been following in other places as well.

Reply:
> That way we'll not be able to keep the product version tracked in the repository and the zip distribution unless rely on git actions/ jenkins builds.

Could you please elaborate on this part? afaiu, git tags has nothing to do with **github actions** or jenkin builds. Git tags are standard construct of git SCM so it wouldn't be a problem to use git tags.